omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -274,7 +274,7 @@
           
           <!-- The "Name" field must match the argument to eventlog.Open() -->
           <util:EventSource Log="Application" Name="DatadogAgent"
-                EventMessageFile="[DIST]agent.exe"
+                EventMessageFile="[BIN]agent.exe"
```

Comment:
`BIN` is the agent binary folder, right? When did it change from `dist` to `bin`?